drivers/pds/lib/utils.go

```diff
@@ -755,6 +757,208 @@ func GetDeploymentCredentials(deploymentID string) (string, error) {
 	return pdsPassword, nil
 }
 
+// This module sets up MySQL Database for Running TPCC. There is some specific requirement that needs to be
+// done for MySQL before running MySQL.
+func SetupMysqlDatabaseForTpcc(dbUser string, pdsPassword string, dnsEndpoint string, namespace string) bool {
+	log.Info("Trying to configure Mysql deployment for TPCC Workload")
+	podSpec := &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "configure-mysql",
+			Namespace:    namespace,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:            "configure-mysql",
+					Image:           pdsTpccImage,
+					Command:         []string{"/bin/sh", "-C", "setup-mysql-for-tpcc.sh", dbUser, pdsPassword, dnsEndpoint},
+					WorkingDir:      "/sysbench-tpcc",
+					ImagePullPolicy: corev1.PullAlways,
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyNever,
+		},
+	}
+	_, err := k8sCore.CreatePod(podSpec)
+	if err != nil {
+		log.Errorf("An Error Occured while creating %v", err)
+		return false
+	}
+
+	//Static sleep to let DB changes settle in
+	time.Sleep(30 * time.Second)
```

Comment:
Hey, the ROI for that is bad here in my opinion. We will have to write a whole function to check 10 seconds of definite required wait time for the pod to come up. Besides, we are further polling the status of the pod right below this. We can still do that but I don't see a lot of value in that. Let me know if you think otherwise - we can come to common conclusion. 
I will simply reduce the sleep to 15 seconds. I kept it 30 seconds to be safe in all honesty, but 15 seconds sleep is enough here. Let me know what you think